sormas-backend/src/main/java/de/symeda/sormas/backend/user/UserService.java

```diff
@@ -186,30 +186,32 @@ public User getByUserName(String userName) {
 		boolean activeOnly,
 		UserRole... userRoles) {
 
-		return getReferenceList(regionUuids, districtUuids, includeSupervisors, filterByJurisdiction, activeOnly, Arrays.asList(userRoles));
+		return getReferenceList(regionUuids, districtUuids, null, includeSupervisors, filterByJurisdiction, activeOnly, Arrays.asList(userRoles));
 	}
 
 	/**
 	 * Loads users filtered by combinable filter conditions.<br />
 	 * Condition combination if parameter is set:<br />
-	 * {@code ((regionUuids & districtUuids & filterByJurisdiction & userRoles) | includeSupervisors) & activeOnly}
-	 * 
+	 * {@code ((regionUuids & districtUuids & communityUuids & filterByJurisdiction & userRoles) | includeSupervisors) & activeOnly}
+	 *
 	 * @see #createJurisdictionFilter(CriteriaBuilder, From)
 	 * @param regionUuids
 	 * @param districtUuids
+	 * @param communityUuids
 	 * @param includeSupervisors
 	 *            If set to {@code true}, all supervisors are returned independent of other filters.
 	 * @param filterByJurisdiction
 	 * @param activeOnly
 	 * @param userRoles
 	 */
 	public List<UserReference> getReferenceList(
-		List<String> regionUuids,
-		List<String> districtUuids,
-		boolean includeSupervisors,
-		boolean filterByJurisdiction,
-		boolean activeOnly,
-		List<UserRole> userRoles) {
+			List<String> regionUuids,
```

Comment:
It appears that the code formatter is configured wrong, since the whole block has a changed intendation.